### PR TITLE
Update minio to use RELEASE.2017-10-27T18-59-02Z

### PR DESCRIPTION
**Pull request by krisis**

### CODE REVIEW

**Comment by joerg84** [comment]
Please retest!

File: repo/packages/M/minio/10/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+               "containerPort":9000,
+               "hostPort":0,
+               "servicePort":{{networking.port}}
+            }
+         ],
+         "privileged":true,
+         "forcePullImage":true
+      }
+   },
+   "healthChecks": [
```
Comment (harshavardhana) [inline repo/packages/M/minio/10/marathon.json.mustache]: Reposting what @joerg84 mentioned on slack. 

https://dcos.io/docs/1.10/deploying-services/creating-services/health-checks/ 
```
{
  "protocol": "COMMAND",
  "command": { "value": "source ./myHealthCheck.sh" }
}
```

We should re-do our healthchecks to use this style.

Comment (joerg84) [reply repo/packages/M/minio/10/marathon.json.mustache]: I ran into that issue with the existing package, if the above hc works I am fine with that.

**Comment by joerg84** [comment]
FYI I just deployed this on a 1.10 cluster and it does not seem to start up:
```
I1102 18:56:42.642662   662 executor.cpp:160] Starting task minio.b7cf41e2-bff7-11e7-bcf5-8a3400f15012.10
I1102 18:58:59.775846   656 executor.cpp:269] Received killTask for task minio.b7cf41e2-bff7-11e7-bcf5-8a3400f15012.10
W1102 18:58:59.775846   655 logging.cpp:91] RAW: Received signal SIGTERM from process 5061 of user 0; exiting```

**Comment by krisis** [comment]
@joerg84 now health check uses COMMAND protocol. I am able to install minio package and run the service on DCOS version 1.10.1.

**Comment by krisis** [comment]
@joerg84 Could you review this PR again?

File: repo/packages/M/minio/10/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "maintainer": "dev@minio.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server for cloud native applications. To deploy at scale, use DC/OS to orchestrate Minio containers one per tenant.",
+  "tags": [
+    "data",
+    "storage",
+    "filesystem"
+  ],
+  "preInstallNotes": "This DC/OS service is currently in preview. There may be bugs, incomplete features, incorrect documentation or other discrepancies in packaging. Preview packages should never be used in production! There is no limit to number of Minio services you can launch. To learn how to install Minio on DC/OS refer to https://github.com/dcos/examples/blob/master/minio/1.9/README.md",
```
Comment (nitisht) [inline repo/packages/M/minio/10/package.json]: @krisis as discussed with @ryadav88 , we can update the `preInstallNotes` to say 

```
This is a community package. Community packages are unverified and unreviewed content from the community.
```

Comment (krisis) [reply repo/packages/M/minio/10/package.json]: @nitisht, @ryadav88  I have updated the preInstallNotes. PTAL

Comment (nitisht) [reply repo/packages/M/minio/10/package.json]: Thanks @krisis LGTM

**Pull request merged**